Bridges/HttpKernel.php

```diff
@@ -161,9 +160,9 @@ protected function mapRequest(ReactRequest $reactRequest)
             if (($type === 'Basic' || $type === 'Digest') && isset($authorizationHeaderParts[1])) {
                 $credentials = base64_decode($authorizationHeaderParts[1]);
                 $credentialsParts = explode(':', $credentials);
-                $_SERVER['PHP_AUTH_USER'] = isset($credentialsParts[0]) ? $credentialsParts[0] : '';
-                $_SERVER['PHP_AUTH_PW'] = isset($credentialsParts[1]) ? $credentialsParts[1] : '';
-                $_SERVER['AUTH_TYPE'] = $type;
+                $headers['PHP_AUTH_USER'] = isset($credentialsParts[0]) ? $credentialsParts[0] : '';
+                $headers['PHP_AUTH_PW'] = isset($credentialsParts[1]) ? $credentialsParts[1] : '';
+                $headers['AUTH_TYPE'] = $type;
             }
```

Comment:
This would be an addon: what about Bearer authentication? It is today in symfony only available from the apache headers. Any idea how to get those across?